src/Exception/GetVariableType.php

```diff
@@ -13,13 +12,10 @@
 
 final class GetVariableType
```

Comment:
Should this whole thing be replaced with `get_debug_type()`?

Reply:
Sounds good. 🙂